src/mame/dec/microvax3100.cpp

```diff
@@ -0,0 +1,170 @@
+#include "cpu/t11/t11.h"
+
+#include "machine/am79c90.h"
+//#include "machine/ncr5380.h"
+//#include "machine/scnxx562.h"
+#include "machine/terminal.h"
+
+
+namespace {
+
+class microvax3100_state : public driver_device
+{
+public:
+	microvax3100_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_terminal(*this, "terminal")
+	{ }
+
+	void microvax3100(machine_config &config);
+
+private:
+	required_device<t11_device> m_maincpu;
+	required_device<generic_terminal_device> m_terminal;
+};
+
+
+// Input ports
+static INPUT_PORTS_START( microvax3100 )
+INPUT_PORTS_END
+
+
+void microvax3100_state::microvax3100(machine_config &config)
+{
+	// Basic machine hardware
+	T11(config, m_maincpu, 66.6667_MHz_XTAL); // Need proper CPU here
+
+	AM7990(config, "lance1", 0); // AMD AM7990PC/80
+
+	// NCR5380(...)
+
+	// DUSCC68562(...) // Signetics SCN68562C4N48
+
+	// Video hardware
+	GENERIC_TERMINAL(config, m_terminal, 0);
+}
+
+ROM_START( mvax3100 )
+	ROM_REGION( 0x40000, "maincpu", ROMREGION_ERASEFF )
+	ROMX_LOAD( "dec89_23-116e8-00_system_rom_lo_word.e24", 0x00000, 0x20000, CRC(69ef8cf5) SHA1(a59a500921278dc356a519cb435641426c844779), ROM_NIBBLE | ROM_SHIFT_NIBBLE_LO | ROM_SKIP(1) )
+	ROMX_LOAD( "dec89_23-117e8-00_system_rom_hi_word.e25", 0x00000, 0x20000, CRC(df572ac3) SHA1(5e91d0f4fc8442e3ebc2424d9f85078ba00d2de5), ROM_NIBBLE | ROM_SHIFT_NIBBLE_HI | ROM_SKIP(1) )
+
+	ROM_REGION( 0x20000, "scsi", ROMREGION_ERASEFF )
+	ROM_LOAD(  "dec89_23-061e8-00_scsi_rom.e98",           0x00000, 0x20000, CRC(51fb8268) SHA1(a930869dce955b9b7a2b0fb68840e863c74e6512) )
+
+	ROM_REGION( 0x10000, "comms", ROMREGION_ERASEFF )
+	ROM_LOAD(  "dec90_fx9123_248e7.bin",                   0x00000, 0x10000, CRC(d50801e6) SHA1(e67b1d732ce775381eb8f41684a3366db7a435d7) )
+
+	ROM_REGION( 0x00117, "plds", ROMREGION_ERASEFF ) // All of them on the comms PCB
+	ROM_LOAD(  "dec90_lm9019_032j7_pal20l10acns.bin",      0x00000, 0x00117, NO_DUMP )
+	ROM_LOAD(  "dec90_lm9027_111l1_pls105anj.bin",         0x00000, 0x00100, NO_DUMP )
+	ROM_LOAD(  "dec90_lm9028_519j5_pal16l8nc.bin",         0x00000, 0x00117, NO_DUMP )
+	ROM_LOAD(  "dec90_lm9029_124l1_pls105anj.bin",         0x00000, 0x00100, NO_DUMP )
+	ROM_LOAD(  "dec90_lm9030_031j7_pal20l10acns.bin",      0x00000, 0x00117, NO_DUMP )
+	ROM_LOAD(  "dec90_lm9031_112l1_pls105anj.bin",         0x00000, 0x00100, NO_DUMP )
+
+	ROM_REGION( 0x00100, "prom", ROMREGION_ERASEFF ) // On the main PCB
+	ROM_LOAD(  "dec84_fx9206_365a1.bin",                   0x00000, 0x00100, NO_DUMP )
+ROM_END
+
+} // anonymous namespace
+
+
+//    YEAR  NAME      PARENT  COMPAT  MACHINE       INPUT         STATE               INIT        COMPANY                          FULLNAME         FLAGS
+COMP( 1989, mvax3100, 0,      0,      microvax3100, microvax3100, microvax3100_state, empty_init, "Digital Equipment Corporation", "MicroVAX 3100", MACHINE_NOT_WORKING | MACHINE_NO_SOUND)
```

Comment:
Unfortunately this is not specific enough, since DEC released several different (though feature-comparable) systems under the "MicroVAX 3100" name. The PCB description suggests that this one is a Model 10 or 20 (there was very little difference between these two).

Reply:
Fixed! It's model 10. We'll dump a Model 20 soon